dd-sdk-android-ndk/src/main/cpp/datadog-native-lib.cpp

```diff
@@ -17,15 +17,30 @@
 #include "utils/datetime.h"
 #include "utils/fileutils.h"
 #include "utils/signal-monitor.h"
+#include "utils/stringutils.h"
+
+static struct rumcontext {
+    char application_id[200];
+    char session_id[200];
+    char view_id[200];
+} rumcontext;
```

Comment:
Can you give a different name to the variable to avoid confusion?

Reply:
Yeah...forgot to change that..is camel best practice for struct names  in C++ ?